v1/css.yaml

```diff
@@ -28,10 +28,12 @@ paths:
         The first two are the same regardless of what domain is used.
```

Comment:
Probably should update this, too. Site is in "the first two". It probably meant to say the first and the third, but probably better to be more explicit here.

Reply:
Updated this text.